fpdf/fpdf.py

```diff
@@ -1976,7 +1938,9 @@ def cell(
         h=None,
         txt="",
         border=0,
-        ln=0,
+        new_x=XPos.RIGHT,
+        new_y=YPos.TOP,
+        ln="DEPRECATED",
         align="",
         fill=False,
         link="",
```

Comment:
Could you also change the default value of the `center` parameter to `"DEPRECATED"`?
And raise a warning in this method, so that we don't pass it anymore to `_render_styled_cell_text()`

Reply:
Ah yes, of course.
The warning gets raised, but we still pass any values on. It's not formally removed yet, after all.